<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatClient_2</class>
 <widget class="QMainWindow" name="ChatClient_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>431</width>
      <height>622</height>
     </rect>
    </property>
    <property name="title">
     <string>Все диалоги</string>
    </property>
    <widget class="QListWidget" name="listWidget_Dialogs">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>431</width>
       <height>571</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_findPeople">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_FindPeople">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>9</y>
      <width>621</width>
      <height>622</height>
     </rect>
    </property>
    <property name="title">
     <string>Диалог</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>601</width>
       <height>561</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>590</y>
       <width>611</width>
       <height>24</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="Send">
        <property name="cursor">
         <cursorShape>SizeHorCursor</cursorShape>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="Exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
